packages/map-template/src/components/Search/Search.jsx

```diff
@@ -139,6 +156,44 @@ function Search({ onLocationClick, onSetSize }) {
         }
     }
 
+    /**
+     * Handle hovering over location.
+     * 
+     * @param {object} location
+     */
+    function onMouseEnter(location) {
+        setHoveredLocation(location);
+    }
+
+    /**
+     * Handle locations clicked on the map.
+     * 
+     * @param {object} location
+     */
+    function onLocationClicked(location) {
+        setCurrentLocation(location);
+
+        // Set the current venue to be the selected location venue.
+        if (location.properties.venueId !== currentVenueName) {
```

Comment:
This highlights a naming discrepancy. We are comparing an "ID" against a "name". Could we with low effort rename `currentVenueName` to `currentVenueId`?

Reply:
Done ✅

Reply:
Ideally the recoil atom should be renamed as well, shouldn't it? (if that doesn't introduce other problems)

Reply:
We decided not to do that.